Source/WebKit/UIProcess/ios/WKExtendedTextInputTraits.mm

```diff
@@ -121,14 +103,35 @@ - (void)setSelectionColorsToMatchTintColor:(UIColor *)tintColor
     static constexpr auto selectionHighlightAlphaComponent = 0.2;
     BOOL shouldUseTintColor = tintColor && tintColor != UIColor.systemBlueColor;
     self.insertionPointColor = shouldUseTintColor ? tintColor : nil;
-#if USE(BROWSERENGINEKIT)
     self.selectionHandleColor = shouldUseTintColor ? tintColor : nil;
-#else
-    self.selectionBarColor = shouldUseTintColor ? tintColor : nil;
-#endif
     self.selectionHighlightColor = shouldUseTintColor ? [tintColor colorWithAlphaComponent:selectionHighlightAlphaComponent] : nil;
 }
 
+- (void)restoreDefaultValues
```

Comment:
Is there any risk if the property list in this method gets out-of-date? (I think no, but want to confirm)

Reply:
Right — it would get out of date if we added a new `@property` to `WKExtendedTextInputTraits`, but didn't update this method as well. That said, I think it's not _that_ easy to make that mistake, as long as whoever's adding the new property searches the existing code for any of the other existing properties, and follows the same precedent.

Reply:
I guess my question is more specifically: "What would the effect be (if any) if the list were to get out-of-date?".

Reply:
Ah, I see. It's pretty unlikely this would user-facing effects even if the list were to get out-of-date, since this would only affect the case where UIKit relies on text input traits outside of text input. The only other trait that falls into this category is selection highlight and handle color (which I made sure to update separately, in this patch), and apparently this `autocorrectionType` which influences whether or not continuous spellchecking is enabled.